.circleci/config.yml

```diff
@@ -105,233 +97,209 @@ jobs:
       - validate-with-maven-script:
-          name: "no-error-test-sbe"
-          image-name: "cimg/openjdk:21.0.6"
-          command: "./.ci/validation.sh no-error-test-sbe"
+          name: no-error-test-sbe
+          image-name: cimg/openjdk:21.0.6
+          command: ./.ci/validation.sh no-error-test-sbe
       - validate-with-maven-script:
-          name: "no-error-spotbugs"
+          name: no-error-spotbugs
           image-name: *cs_img
-          command: "./.ci/validation.sh no-error-spotbugs"
-      - validate-with-maven-script:
-          name: "no-error-pgjdbc"
-          image-name: "cimg/openjdk:21.0"
-          command: "./.ci/validation.sh no-error-pgjdbc"
+          command: ./.ci/validation.sh no-error-spotbugs
       - validate-with-maven-script:
-          name: "linkcheck-plugin"
-          image-name: "cimg/openjdk:21.0.6"
-          command: "./.ci/run-link-check-plugin.sh --skip-external"
+          name: no-error-pgjdbc
+          image-name: cimg/openjdk:21.0
+          command: ./.ci/validation.sh no-error-pgjdbc
       - validate-with-maven-script:
-          name: "no-exception-samples-gradle"
-          image-name: "cimg/openjdk:17.0.7"
-          command: "./.ci/no-exception-test.sh no-exception-samples-gradle"
+          name: no-exception-samples-gradle
+          image-name: cimg/openjdk:17.0.7
+          command: ./.ci/no-exception-test.sh no-exception-samples-gradle
       - validate-with-maven-script:
-          name: "no-exception-samples-maven"
-          image-name: "cimg/openjdk:21.0.6"
-          command: "./.ci/no-exception-test.sh no-exception-samples-maven"
-
+          name: no-exception-samples-maven
+          image-name: cimg/openjdk:21.0.6
+          command: ./.ci/no-exception-test.sh no-exception-samples-maven
       - validate-with-maven-script:
-          name: "no-exception-samples-ant"
+          name: no-exception-samples-ant
           image-name: *cs_img
-          command: "./.ci/no-exception-test.sh no-exception-samples-ant"
+          command: ./.ci/no-exception-test.sh no-exception-samples-ant
       - validate-with-maven-script:
-          name: "no-error-hazelcast"
+          name: no-error-hazelcast
           image-name: *cs_img
-          command: "./.ci/validation.sh no-error-hazelcast"
-
+          command: ./.ci/validation.sh no-error-hazelcast
   idea:
     jobs:
       - run-inspections
-
   git-validation:
     jobs:
       - validate-with-script:
-          name: "git-no-merge-commits"
-          command: "./.ci/validation.sh git-no-merge-commits"
+          name: git-no-merge-commits
+          command: ./.ci/validation.sh git-no-merge-commits
       - validate-with-script:
-          name: "git-check-pull-number"
-          command: "./.ci/validation.sh git-check-pull-number"
-
+          name: git-check-pull-number
+          command: ./.ci/validation.sh git-check-pull-number
   cli-validation:
     jobs:
-      - yamllint
```

Comment:
you asked how it would be simpler imho if possible this would help: With the colon `:` it has same style and does not look lost anymore.

<img width="1182" height="367" alt="Image" src="https://github.com/user-attachments/assets/6ee485b6-0ec6-4f35-89db-e711f4b3732d" />